CHANGELOG.md

```diff
@@ -34,6 +34,7 @@
 - Minor: Added whitespace trim to username field in nicknames (#3946)
 - Minor: Added `Go to message` context menu action to search popup, mentions, usercard and reply threads. (#3953)
 - Minor: Added link back to original message that was deleted. (#3953)
+- Minor: Input text is retained when replying to a message. (#3989)
```

Comment:
We could probably add this to the existing reply threads changelog entry with 3722.

Reply:
Agreed

Reply:
True. I'd say this doesn't really need an entry at all since there isn't a release between this PR and the original one.

Reply:
I went with adding this PR to the existing changelog entry